introduction/Charles/about.md

```diff
@@ -0,0 +1,13 @@
+ # Introduction
```

Comment:
Why are these implemented in this PR, if this PR is only supposed to solve issue #47